io.github.realkleiner.yafc-ce.metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2025 realkleiner -->
+<component type="desktop-application">
+  <id>io.github.realkleiner.yafc-ce</id>
```

Comment:
I'm confused why is this under your namespace?

The app is at https://github.com/shpaass/yafc-ce.git

Your repo has manifest and metainfo files.